build_tools/therock_build_observability.py

```diff
@@ -0,0 +1,390 @@
+#!/usr/bin/env python3
+"""
+author: shiraz.ali@amd.com
```

Comment:
We haven't been including individual authorship in files. It's there in the git commit history already.